addons/event_booth/models/event_type_booth.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventBooth(models.Model):
+    _name = 'event.type.booth'
+    _description = 'Event Booth Template'
+
+    def _get_default_booth_category(self):
+        categories = self.env['event.booth.category'].search([])
+        if categories and len(categories) == 1:
+            return categories
```

Comment:
add note : assign category by default if only one exists
and postfix by _id to get the complete field_name